share/revng/test/tests/abi/prepare-tested-model.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import argparse
```

Comment:
What does this script do? Add brief summary on top.

Reply:
It's just `replace-abi` script with a bit of extra filtering on top.

Reply:
OK, but add a brief summary.

Reply:
Done:
```
# This script is used for model preparation for the tests using runtime-abi
# artifacts. It removes all the function not explicitly relevant to the test
# so only `test_X` function plus a couple of helpers (like `main` and `memcpy`)
# remain. After which, it ensures that the prototypes of all the remaining
# functions have their ABI set correctly to work around the limitations of
# ABI detection when importing from dwarf.
```